auth-web/.env.example

```diff
@@ -18,6 +18,7 @@ VUE_APP_XLP_FORMS_URL="https://www2.gov.bc.ca/assets/gov/employment-business-and
 
 #vaults web-url
 VUE_APP_REGISTRY_HOME_URL="https://dev.bcregistry.gov.bc.ca/"
+VUE_APP_REGISTRY_URL="https://dev.bcregistry.gov.bc.ca"
```

Comment:
Isn't this the same as the line above?

Reply:
from PROD:

VUE_APP_REGISTRY_HOME_URL="https://www.bcregistry.gov.bc.ca/"

Reply:
thanks for point it out, it's same, my local .env HOME_URL direct to https://dev.bcros.ca/, which is wrong, I'll update it.